packages/block-library/src/cover/edit.native.js

```diff
@@ -330,27 +348,45 @@ const Cover = ( {
 		<View style={ styles.backgroundContainer }>
 			{ controls }
 
+			{ openMediaOptionsRef?.current && (
+				<View style={ styles.imageEditButton }>
+					<ImageEditingButton
+						isSelected={ isParentSelected }
+						isUploadFailed={ didUploadFail }
+						isUploadInProgress={ isUploadInProgress }
+						onSelectMediaUploadOption={ onSelectMedia }
+						openMediaOptions={ openMediaOptionsRef?.current }
```

Comment:
I don't think we need the optional chaining for `openMediaOptionsRef`, it will always exist (from `useRef`). Wdyt?

Reply:
You're right! I'll remove the `?.` thanks!